source/access/generating_keys_with_openssh.rst

```diff
@@ -57,25 +57,28 @@ this particular case, the private key is ``id_rsa`` and public key is
 ``id_rsa.pub``. You may have multiple keys (not necessarily in the
 directory ``~/.ssh``) if you or your operating system requires this.
 
+.. warning::
+
+   For security reasons, users should always generate a new key pair for use in
+   the VSC clusters, and only use it for the VSC clusters.
+
 You will need to generate a new key pair, when:
 
--  you don't have a key pair yet, or
--  you have a key pair, but is not in the correct format (RSA) or
-   uses too few bits (at least 4096),
+-  you don’t yet have a key pair that is dedicated for the VSC clusters, or
 -  you forgot the passphrase protecting your private key, or
 -  or your private key was compromised.
 
-To generate a new public/private pair, use the following command:
+To generate a new public/private pair, use the following command (make sure to
+generate a 4096-bit key):
 
 ::
 
-   $ ssh-keygen -t rsa -b 4096
+   $ ssh-keygen -t rsa -b 4096 -f ~/.ssh/id_rsa_vsc
```

Comment:
This change may have implications for other parts of the documentation, since the relevant private key will not be the default one.

Reply:
@gjbex which parts are you thinking of specifically?

Reply:
@smoors The page(s) where the keys are used to connect.  -i ~/.ssh/id_rsa_vsc would have to be added, I guess.

Reply:
@gjbex this can be avoided by linking the key to the vsc-id
https://github.com/hpcleuven/VscDocumentation/pull/282/commits/d5371aaad2fd4eea1c109f067c7168cd584aabc5